docker_entrypoint.sh

```diff
@@ -40,189 +62,304 @@ ADV_TOR_MAXSOCKSCONNS=$(yq e '.advanced.tor.maxsocksconns' ${BITMONERO_DIR}/star
 fi
 if [ "$ADV_TOR_RPCBAN" = "false" ] ; then
- disable_rpc_ban="disable-rpc-ban=1              # Do not ban hosts on RPC errors. May help to prevent monerod from banning traffic originating from the Tor daemon."
- echo -e "\n# RPC Ban\n$disable_rpc_ban" >> $new_conf_template
+ disable_rpc_ban="disable-rpc-ban=1              # Do not ban hosts on RPC errors. May help prevent monerod from banning traffic originating from the Tor daemon."
+ echo -e "\n# RPC Ban\n$disable_rpc_ban" >> $config_file
 fi
 
 #TOR config:
 if [ "$ADV_TOR_TORONLY" = "true" ] ; then
- echo -e "\n# TOR" >> $new_conf_template
- echo    "# Proxy for broadcasting/relaying transaction (does not fetch blocks)" >> $new_conf_template
- echo -n "tx-proxy=tor,TOR_HOSTNAME:TOR_PORT,ADV_TOR_MAXSOCKSCONNS" >> $new_conf_template
+ echo -e "\n# TOR" >> $config_file
+ echo    "# Proxy for broadcasting/relaying transaction (does not fetch blocks)" >> $config_file
+ echo -n "tx-proxy=tor,TOR_SOCKS_PROXY_HOST:TOR_PORT,ADV_TOR_MAXSOCKSCONNS" >> $config_file
  if [ "$ADV_TOR_DANDELION" = "false" ] ; then
-  echo ",disable_noise" >> $new_conf_template
+  echo ",disable_noise" >> $config_file
  fi
- echo -e "\n# Use Tor's socks proxy for p2p traffic (note: --proxy cannot reach .onion nodes)" >> $new_conf_template
- echo    "proxy=TOR_HOSTNAME:TOR_PORT" >> $new_conf_template
- echo    "# Pad relayed transactions to next 1024 bytes to help defend against traffic volume analysis. This only makes sense if you are behind Tor or I2P." >> $new_conf_template
- echo    "pad-transactions=1" >> $new_conf_template
+ echo -e "\n# Use Tor's socks proxy for p2p traffic (note: --proxy cannot reach .onion nodes)" >> $config_file
+ echo    "proxy=TOR_SOCKS_PROXY_HOST:TOR_PORT" >> $config_file
+ echo    "# Pad relayed transactions to next 1024 bytes to help defend against traffic volume analysis. This only makes sense if you are behind Tor or I2P." >> $config_file
+ echo    "pad-transactions=1" >> $config_file
 fi
 
 #Gossip config:
 if [ "$ADV_P2P_GOSSIP" = "false" ] ; then
- echo -e "\n# GOSSIP" >> $new_conf_template
- echo "#Tell our peers not to gossip our node" >> $new_conf_template
- echo "hide-my-port=1" >> $new_conf_template
- echo "# Disable UPnP port mapping" >> $new_conf_template
- echo "igd=disabled" >> $new_conf_template
-elif [ "$ADV_P2P_PUBLICRPC" = "true" ] ; then
+ echo -e "\n# GOSSIP" >> $config_file
+ echo "#Tell our peers not to gossip our node w/ P2P port" >> $config_file
+ echo "hide-my-port=1" >> $config_file
+ echo "# Disable UPnP port mapping" >> $config_file
+ echo "igd=disabled" >> $config_file
+elif [ "$ADV_TOR_TORONLY" = "true" ] ; then
+  echo "# Advertise our onion as the reachable host for incoming P2P connections" >> $config_file
+  echo "anonymous-inbound=PEER_TOR_ADDRESS:MONERO_P2P_PORT,MONEROD_LOCAL_HOST:MONERO_P2P_PORT_LOCAL_BIND,ADV_TOR_MAXONIONCONNS" >> $config_file
```

Comment:
```suggestion
  echo "anonymous-inbound=PEER_TOR_ADDRESS:MONERO_P2P_PORT,MONEROD_LOCAL_HOST:MONERO_P2P_PORT_LOCAL_BIND,ADV_TOR_MAXONIONCONNS" >> $config_file
```
use new port, 18084, not MONERO_P2P_PORT

Reply:
18084 for the local bind port it is...